components/apimgt/org.wso2.carbon.apimgt.tracing/src/main/java/org/wso2/carbon/apimgt/tracing/Util.java

```diff
@@ -0,0 +1,130 @@
+            } else {
+                childSpan = tracer.getTracingTracer().buildSpan(spanName).asChildOf((SpanContext) sp).start();
+            }
+            return new TracingSpan(childSpan);
+        }
+    }
+
+    public static void setTag(TracingSpan span, String key, String value) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            ((Span) sp).setTag(key, value);
+        }
+    }
+
+    public static void setLog(TracingSpan span, String key, String value) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            ((Span) sp).log(ImmutableMap.of(key, value));
+        }
+    }
+
+    public static void setLog(TracingSpan span, String key) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            ((Span) sp).log(key);
+        }
+    }
+
+    public static void finishSpan(TracingSpan span) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            ((Span) sp).finish();
+        }
+    }
+
+    public static void inject(TracingSpan span, TracingTracer tracer, Map<String, String> tracerSpecificCarrier) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            tracer.getTracingTracer().inject(((Span) sp).context(), Format.Builtin.HTTP_HEADERS,
+                    new TextMapInjectAdapter(tracerSpecificCarrier));
+        } else if (sp instanceof SpanContext) {
+            tracer.getTracingTracer().inject((SpanContext) sp, Format.Builtin.HTTP_HEADERS,
+                    new TextMapInjectAdapter(tracerSpecificCarrier));
+        }
+    }
+
+    public static TracingSpan extract(TracingTracer tracer, Map<String, String> headerMap) {
+        return new TracingSpan(tracer.getTracingTracer().extract(Format.Builtin.HTTP_HEADERS,
+                new TextMapExtractAdapter(headerMap)));
+    }
+
+    public static TracingTracer getGlobalTracer() {
+        return new TracingTracer(GlobalTracer.get());
+    }
+
+    public static void baggageSet(TracingSpan span, String key, String value) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            ((Span) sp).setBaggageItem(key, value);
+        }
+    }
+
+    public static String baggageGet(TracingSpan span, String key) {
+        Object sp = span.getSpan();
+        if (sp instanceof Span) {
+            return ((Span) sp).getBaggageItem(key);
+        } else if (sp instanceof SpanContext) {
+            Iterable<Map.Entry<String, String>> entries = ((SpanContext) sp).baggageItems();
+            for (Map.Entry<String, String> entry : entries) {
+                if (entry.getKey().equals(key)) {
+                    return entry.getValue();
+                }
+            }
+        }
+        return null;
+    }
+
+    public static void setTracingEnabled(boolean isTraceEnabled){
+        Util.isTraceEnabled = isTraceEnabled;
```

Comment:
you can directly access the variable as `isTraceEnabled`. No need to access it as `Util.isTraceEnabled`